Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -124,6 +124,26 @@
     pointSize : "5"
 });
 
+addStyle('Sign', {
+    color : "color() * sign(${temperature})",
+    pointSize : "5"
+});
```

Comment:
Since `temperature` is always positive this style won't look very special. One idea could be using the sign of the ${POSITIONS} values (like the tests below).